examples/basic_usage/README.md

```diff
@@ -12,13 +12,12 @@ Note that this example may create resources which cost money. Run
 
 | Name | Version |
 |------|---------|
-| terraform | ~> 0.14.0 |
+| terraform | ~> 1.0 |
+| aws | ~> 3.38 |
 
 ## Providers ##
 
-| Name | Version |
-|------|---------|
-| aws | ~> 3.38 |
+No providers.
```

Comment:
See [this discussion](https://github.com/cisagov/ami-build-iam-user-tf-module/pull/34#pullrequestreview-760922369).